src/ValueObject/Application/File.php

```diff
@@ -152,4 +154,14 @@ public function getRectorWithLineChanges(): array
     {
         return $this->rectorWithLineChanges;
     }
+
+    public function hasInlineHTMLNode(): bool
+    {
```

Comment:
This will broke many apps unfortunately, the skip inline html if needed, is only when near of it appended/prepended/removed, the Stmt that not InlineHTML when changed should working fine, so better show symfony warning instead of Exception, I will create a PR for it.

Reply:
https://github.com/rectorphp/rector-src/pull/4058